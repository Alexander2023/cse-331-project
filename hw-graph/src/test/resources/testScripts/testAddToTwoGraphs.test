# Create the graphs
CreateGraph graph1
CreateGraph graph2

# Add a node to each
AddNode graph1 n1
AddNode graph2 n2

# Print the children of the graphs
ListNodes graph1
ListNodes graph2