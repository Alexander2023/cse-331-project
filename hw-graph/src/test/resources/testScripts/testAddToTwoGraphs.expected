# Create the graphs
created graph graph1
created graph graph2

# Add a node to each
added node n1 to graph1
added node n2 to graph2

# Print the children of the graphs
graph1 contains: n1
graph2 contains: n2